apps/openchallenges/challenge-service/src/main/resources/db/challenges.csv

```diff
@@ -503,7 +503,7 @@
-"505","mario","🕹️ 🍄 MARIO : Monitoring AMD progression in OCT","Improve the planning of anti-VEGF treatments","Age-related Macular Degeneration (AMD) is a progressive degeneration of the macula, the central part of the retina, affecting nearly 196 million people worldwide 1. It can appear from the age of 50, and more frequently from the age of 65 onwards, causing a significant weakening of visual capacities, without destroying them. It is a complex and multifactorial pathology in which genetic and environmental risk factors are intertwined. Advanced stages of the disease (atrophy and neovascularization) affect nearly 20% of patients: they are the first cause of severe visual impairment and blindness in developed countries. Since their introduction in 2007, Anti–vascular endothelial growth factor (anti-VEGF) treatments have proven their ability to slow disease progression and even improve visual function in neovascular forms of AMD 2. This effectiveness is optimized by ensuring a short time between the diagnosis of the pathology and the start of treatment as well as by performing regular ch...","https://rumc-gcorg-p-public.s3.amazonaws.com/logos/challenge/666/square_image_mario_t8tUYoc.png","https://mario.grand-challenge.org/","active","5","","2024-04-01","2024-07-10","\N","2024-04-29 18:13:15","2024-04-29 18:13:26"
+"505","🕹️-🍄-mario--monitoring-amd-progression-in-oct","🕹️ 🍄 MARIO : Monitoring AMD progression in OCT","Improve the planning of anti-VEGF treatments","Age-related Macular Degeneration (AMD) is a progressive degeneration of the macula, the central part of the retina, affecting nearly 196 million people worldwide 1. It can appear from the age of 50, and more frequently from the age of 65 onwards, causing a significant weakening of visual capacities, without destroying them. It is a complex and multifactorial pathology in which genetic and environmental risk factors are intertwined. Advanced stages of the disease (atrophy and neovascularization) affect nearly 20% of patients: they are the first cause of severe visual impairment and blindness in developed countries. Since their introduction in 2007, Anti–vascular endothelial growth factor (anti-VEGF) treatments have proven their ability to slow disease progression and even improve visual function in neovascular forms of AMD 2. This effectiveness is optimized by ensuring a short time between the diagnosis of the pathology and the start of treatment as well as by performing regular ch...","https://rumc-gcorg-p-public.s3.amazonaws.com/logos/challenge/666/square_image_mario_t8tUYoc.png","https://www.codabench.org/competitions/2851/","active","10","","2024-04-01","2024-07-10","\N","2024-04-29 18:13:15","2024-05-09 18:33:40"
```

Comment:
The challenge updated is not inserted into the DB. I didn't see any error messages but I confirmed that there are two different issues that prevent the record from being inserted:

- the challenge slug must not include emoticons
- the challenge slug must not include more than one consecutive dash (i.e. "--" is not valid)

Reply:
> there are two different issues that prevent the record from being inserted

Ah yes, this was caught in the Google sheet as well (highlighted yellow for invalid formatting):

![Screenshot 2024-05-10 at 9 28 24 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/40383a0d-77a8-4784-b7ff-e1cf797fbb64)

I will update this

Reply:
Updated ✅